framework/include/base/Assembly.h

```diff
@@ -2257,6 +2257,8 @@ class Assembly
   mutable std::map<unsigned int, std::map<FEType, FEVectorBase *>> _vector_fe_lower;
   /// Vector FE objects for lower dimensional elements
   mutable std::map<unsigned int, std::map<FEType, const FEVectorBase *>> _const_vector_fe_lower;
+  /// helper object for transforming coordinates for lower dimensional element quadrature points
+  std::map<unsigned int, FEBase **> _holder_fe_lower_helper;
```

Comment:
Why not a vector or even `std::array<FEBase **, 2>` ? I think fixed size storage is pretty perfect here

Reply:
I think you could make an argument for the currently used container and both containers you proposed. The size this container *needs* to be is not fixed; it is equal to `_mesh_dimension`. So you could wind up with wasted space in your `std::array` (that no one would ever care about). A vector makes logical sense since you're index perfectly corresponds to dim. Maybe a follow-on PR could change all these `_holder_fe_*helper` objects to `std::vector`

Reply:
> Maybe a follow-on PR could change all these

In typical MOOSE fashion 😆

Reply:
It's not as if anything is broken or wrong here. And cosmetic changes would be more attractive if it didn't force re-triggering a few hour testing suite :smile:

Reply:
To be clear: I'm not disapproving and understand 😬